upload-cloud-storage/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
The tests are using mocha/chai, so we shouldn't need this jest config

Reply:
done